homeassistant/components/motion_blinds/__init__.py

```diff
@@ -137,6 +136,11 @@ def stop_motion_multicast(event):
         KEY_COORDINATOR: coordinator,
     }
 
+    if motion_gateway.firmware is not None:
+        version = f"{motion_gateway.firmware}, proto: {motion_gateway.protocol}"
+    else:
+        version = motion_gateway.protocol
```

Comment:
It looks weird to only include `"proto:"` in one of these cases. I'd make this consistent. Maybe have two prefixes, one for firmware and one for protocol?

Reply:
The motion_gateway.firmware actually already includes "Firmware: " in the response received from the gateway (weirdly enough) so now it looks like:
`Firmware: A1.0.1_B0.1.4, proto: 0.9`
I will change both to have prefix "protocol"

Reply:
done